addons/binding/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/handler/OnkyoHandler.java

```diff
@@ -720,6 +778,7 @@ private void checkStatus() {
         sendCommand(EiscpCommand.POWER_QUERY);
 
         if (connection != null && connection.isConnected()) {
+
```

Comment:
Does this empty line nowadays cause checkstyle warning?

Reply:
Not sure if it does - this was accidental anyways, I'll remove it.